packages/package.go

```diff
@@ -401,6 +401,32 @@ func (p *Package) WorksWithCapabilities(capabilities []string) bool {
 	return true
 }
 
+func (p *Package) HasCompatibleSpec(specMin, specMax, kibanaVersion *semver.Version) (bool, error) {
+	if specMin == nil && specMax == nil {
+		return true, nil
+	}
+	if specMin == nil && kibanaVersion == nil {
+		specMin = semver.MustParse("3.0.0")
+	}
```

Comment:
Not totally sure about this, this could be changed depending on what it is decided here https://github.com/elastic/package-registry/issues/1050#issuecomment-1643650408

Reply:
Removed this default minimum spec.
So in case there is no `spec.min` in the default query, it will return all the packages filtered just by `spec.max`